pkgs/top-level/perl-packages.nix

```diff
@@ -6166,6 +6178,18 @@ let self = _self // overrides; _self = with self; {
   # For backwards compatibility.
   if_ = self."if";
 
+  ImageInfo = buildPerlPackage rec {
+    name = "Image-Info-1.38";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/S/SR/SREZIC/${name}.tar.gz";
+      sha256 = "06g9nwp9k3arsma09qxkxlxsc5y92kzyjvwmczvylpsmc5b8p9mq";
+    };
+    meta = {
+      description = "Extract meta information from image files";
+      license = "perl";
```

Comment:
Instead of
```
license = "perl"
```
prefer
```
license = with stdenv.lib.licenes; [ artistic1 gpl1 ];
```
Did you use the `nix-generate-from-cpan` tool? When I use that tool it also adds `perlPackages.IOstringy` as a runtime dependency of ImageInfo.